### DISCUSSION TITLE
Postgres 9.5 Support

### DISCUSSION THREAD

**Discussion by pvkarthik-coder** [testcontainers/testcontainers-java#5352] [Ideas]
Hi, Is there a version of testcontainer that supports Postgres 9.5 and above? If not it should be on the roadmap

**Comment by eddumelendez** [comment]
Hi @pvkarthik-coder 
Which kind of support are you looking for? Currently, `PostgresContainer` works by default with `9.6.12`. If you want to use another version you can do something like this `new PostgreSQLContainer<>("postgres:10.20")`
